base/ubi8-python-3.8/Dockerfile

```diff
@@ -1,4 +1,21 @@
-FROM registry.access.redhat.com/ubi8/python-38:latest
+FROM registry.access.redhat.com/ubi8/ubi:latest
```

Comment:
Can we use `ubi-minimal` instead?

Reply:
That would either mean living with microdnf, or installing regular dnf first thing, at which points the benefits or the different ubi base are pretty much lost, me thinks.

In any case, I'd want to do such large changes in multiple steps, across multiple PRs.

So far, I don't even have the modest change in this PR groomed. It's necessary to move slowly and cautiously, and to build consensus.

The disk savings are small, 300mb across three base images.

Reply:
/retest